apiserver/facades/client/storage/storage.go

```diff
@@ -1111,3 +1134,21 @@ func (a *APIv4) importFilesystem(
 
 // Destroy was dropped in V4, replaced with Remove.
 func (*APIv4) Destroy(_, _ struct{}) {}
+
+// DeletePool deletes the named pool
+func (a *APIv5) DeletePool(poolName string) error {
```

Comment:
You'd need to mask these on V4 if you change the embedding to the current approach